lib/app/console_app.rb

```diff
@@ -129,13 +129,13 @@ def connector_registered?(warn_if_not: true)
     end
 
     def create_connector(index_name, force: false)
-      connector_settings = Core::ConnectorSettings.fetch(connector_id)
-
-      if connector_settings.nil? || force
-        created_id = Core::ElasticConnectorActions.create_connector(index_name, App::Config['service_type'])
-        connector_settings = Core::ConnectorSettings.fetch(created_id)
+      id = connector_id
+      if Core::ElasticConnectorActions.get_connector(connector_id)[:found] == false || force
```

Comment:
`== false` is not something you want to keep. never use a bool after `==`

you want:

```
unless Core::ElasticConnectorActions.get_connector(connector_id)[:found]  || force
```
(untested)

Reply:
Right, I should have changed `||` to `&&`

```
if !Core::ElasticConnectorActions.get_connector(connector_id)[:found] || force
```
or 

```
unless Core::ElasticConnectorActions.get_connector(connector_id)[:found] && force
```

but you get the idea